crates/sc-proof-of-time/src/source.rs

```diff
@@ -46,7 +48,7 @@ pub struct PotSlotInfo {
 
 /// Stream with proof of time slots
 #[derive(Debug, Deref, DerefMut)]
-pub struct PotSlotInfoStream(mpsc::Receiver<PotSlotInfo>);
+pub struct PotSlotInfoStream(Receiver<PotSlotInfo>);
```

Comment:
This is fine, but I tend to leave `mpsc::` or `broadcast::` because it is very common to also have `oneshot` or other channels involved in the same context, so it helps with readability.